### DISCUSSION TITLE
接类zzz影画制作（低技术力）

### DISCUSSION THREAD

**Discussion by jjy2022** [share121/inter-knot#2522] [其他]
![爱丽第一版](https://github.com/user-attachments/assets/5050c1c3-7ca5-4632-b2ea-c64dfb363c13)
如图
![bd9e83d96c3470cdcfa53566529b8c3ba68dfd8a-1920x1080](https://github.com/user-attachments/assets/3cee2763-8b1c-4ece-9bad-f0c962d97f84)
留言就考虑做（本人学生产能低下）

**Comment by guozichang2007** [comment]
太帅辣<img src="https://upload-bbs.mihoyo.com/upload/2022/09/20/80a7ef5a067a1acd495fcc09140b069c_5444302550606416083.gif" alt="爱莉希雅-比心" width="75" >

**Reply by gold-shooter** [reply]
PS弄的？

**Reply by jjy2022** [reply]
对
